features/openhab-addons-verify/pom.xml

```diff
@@ -45,6 +45,13 @@
             <type>xml</type>
         </dependency>
         <dependency>
+            <groupId>org.openhab.core</groupId>
+            <artifactId>openhab-esh-addons</artifactId>
```

Comment:
To get this to be enabled, I had to manually add:
```
   <descriptor>mvn:org.openhab.core/openhab-esh-addons/${ohc.version}/xml/features</descriptor>
```

to the `descriptors` block below.  Without it, the repo entry was missing and the MiOS Binding wouldn't resolve the `openhab-transformation-map` feature dependency.

Not sure if that's correct or not...